src/Serializer/Normalizer/PhoneNumberNormalizer.php

```diff
@@ -101,6 +101,14 @@ public function denormalize($data, $class, $format = null, array $context = []):
      */
     public function supportsDenormalization($data, $type, $format = null, array $context = []): bool
```

Comment:
Adding a bc notice here would be nice don't you think?

Reply:
not sure I follow. BC notice for what exactly?

Reply:
I don't know why I ended up with the conclusion that supportsDenormalization is deprecated, but not at all 🤔 .